.ci/static-checks.sh

```diff
@@ -760,10 +760,13 @@ static_check_docs()
 	# Now, spell check the docs
 	cmd="${tests_repo_dir}/cmd/check-spelling/kata-spell-check.sh"
 
+	local docs_failed=0
 	for doc in $docs
 	do
-		"$cmd" check "$doc" || die "spell check failed for document $doc"
+		"$cmd" check "$doc" || { info "spell check failed for document $doc" && docs_failed=1; }
```

Comment:
we could increment the variable value if we want to tell the user how many errors he got, but if not, I think this way is fine.

Reply:
I did originally think of increment, but then ran away from the idea of adding any `bc` or `(( ))` code into a single line expression, so went with the simple `=1` option. We could also collect up the names of the `$doc`s that failed, and list them at the end.

I can add those improvements if we want. I think as it is, all the necessary info should appear in the logs fairly grouped together.
Somebody just shout if they want the improvements adding.